lib/datadog/appsec.rb

```diff
@@ -21,3 +21,7 @@ def self.writer
 require_relative 'appsec/contrib/rack/integration'
 require_relative 'appsec/contrib/sinatra/integration'
 require_relative 'appsec/contrib/rails/integration'
+
+# Internal appsec events monitor
+require_relative 'appsec/monitor'
+Datadog::AppSec::Monitor::Gateway::Watcher.watch
```

Comment:
Probably this should be plugged into the AppSec component soon, notably to condition this behind AppSec being enabled.